### ISSUE TITLE
A11y_Planner_Plan Chart page_ Screen reader: Unable to select the 'on track', 'at risk' and 'blocked' bar graph with talkback in bucket bar graph pane. 

### ISSUE THREAD

**Issue by msftedad** [PhilJay/MPAndroidChart#5455]
**Summary**

Repro Steps: 
 1. Using the url: https://github.com/PhilJay/MPAndroidChart/blob/master/MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/StackedBarActivity.java, emulator the app using any Android emulator.
2. Turn on Talkback.
3. In Stacked Bar activity screen, try to access different bars using talkback.

https://github.com/PhilJay/MPAndroidChart/assets/93735775/62a4c372-1e4d-4a95-9a0d-d7b956603ed8

**Actual Results**
User is unable to select the on track, at risk and blocked bar graph with talkback in bucket bar graph pane.

**Expected Behavior**

User should be able to select the on track, at risk and blocked bar graph and in track with talkback.

**Possible Solution**

**Device (please complete the following information):**
 - Device: [e.g. Google Pixel]
 - Android Version: 12
 - Library Version (e.g. 3.1.0-alpha)

**Additional Context**

ADD A REWARD using [Speed](speedoss.com) to SOLVE this issue QUICKLY and SUPPORT this project.

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
